docs/modules/ROOT/pages/vshn-managed/postgresql/restore.adoc

```diff
@@ -0,0 +1,78 @@
+= Restore a PostgreSQL service
+
+IMPORTANT: The restores can only be triggered from an instance in the same namespace.
+If the old instance has already been deleted the backups are deleted as well.
+
+. Get the name of the backup you want to restore by listing all available backups.
++
+[source,bash]
+----
+kubectl -n postgres get vshnpostgresbackups
+BACKUP NAME                                 DATABASE INSTANCE     STATUS     AGE
+pgsql-app1-prod-pk8k4-2023-03-05-13-05-00   pgsql-app1-prod       Completed  38s
+pgsql-app1-dev-pk8k4-2023-03-01-16-52-11    pgsql-app1-dev        Failed     28s
+----
+NOTE: The list will show all backups from all instances in this namespace. In the example above there are two database instances with one backup each.
+
+. Choose a specific backup and get its details.
```

Comment:
For what do I need the details exactly?

Reply:
This step is not necessary for the subsequent steps but it's a way to tell the customer he can have more details regarding its backups.

Reply:
Then please add it to a separate section. Step-by-step tutorials should come to a point, not adding any unnecessary detours.

Reply:
I added a new page as it seemed to me it's not a good idea to show how backups work in a restore page.